alerter.c

```diff
@@ -23,10 +23,17 @@ void alertInCelcius(float farenheit) {
     }
 }
 
-int main() {
-    alertInCelcius(400.5);
-    alertInCelcius(303.6);
+void testalertInCelcius()
+{
+    alertInCelcius(FRENHEIT_VALUE1);
+    assert(alertFailureCount == FALSE);
+    alertInCelcius(FRENHEIT_VALUE2);
+    assert(alertFailureCount == TRUE);

```

Comment:
There is a bug in the production code. That bug should be caught by the test code. As of now without modifying the production code, we are supposed to write test cases that defines the expected outcome from production code. Whether the production code returns right value is secondary now. Primary point is to strengthen test cases.

Hence AlertFailure count is a counter that tells us how many alerts have been raised. So if you have input 1 values below threshold and 2 values above threshold, the count should be 2.
Please update the tests.